### ISSUE TITLE
createJSON function does not prevent base64 encoding

### ISSUE THREAD

**Issue by szymon-musiol** [majst01/fluent-bit-go-redis-output#5]
A function createJSON has a code which should handle base64 encoding of data but it does not work for all cases.

The function is located here: https://github.com/majst01/fluent-bit-go-redis-output/blob/0d65f97be83df4e8f44a7a57ffc2c124a922ed55/out_redis.go#L155-L174

But for example data:
```
	mapInterface := map[interface{}]interface{}{
		"annotations": map[interface{}]interface{}{
			"checksum/config": [13]int{111, 98, 115, 101, 114, 118, 97, 98, 105, 108, 105, 116, 121},
			"namespace_name":  [13]int{111, 98, 115, 101, 114, 118, 97, 98, 105, 108, 105, 116, 121},
		},
	}
```

it will produce base64 encoding in values for keys "checksum/config", "namespace_name".

Recursive call would be needed here.

Such cases will occur often, it is enough to enable fluent-bit kubernetes filter plugin.

**Comment by szymon-musiol** [comment]
The fix have bee done here https://github.com/majst01/fluent-bit-go-redis-output/pull/6

**Comment by majst01** [comment]
closed with #8
